core/src/main/java/tachyon/client/TachyonFS.java

```diff
@@ -89,28 +93,49 @@ public static synchronized TachyonFS get(final TachyonURI tachyonURI) throws IOE
         throw new IOException("Invalid Tachyon URI: " + tachyonURI + ". Use " + Constants.HEADER
             + "host:port/ ," + Constants.HEADER_FT + "host:port/");
       }
-      return new TachyonFS(tachyonURI);
+
+      boolean useZookeeper = scheme.equals(Constants.SCHEME_FT);
+      tachyonConf.set(Constants.USE_ZOOKEEPER, Boolean.toString(useZookeeper));
+      tachyonConf.set(Constants.MASTER_HOSTNAME, tachyonURI.getHost());
+      tachyonConf.set(Constants.MASTER_PORT, Integer.toString(tachyonURI.getPort()));
```

Comment:
Are the sets just for the case when the user does not update the new tachyon properties files?